locales/en.yml

```
@@ -1,3 +1,19 @@
+yes: 'Yes'
+no: 'No'
+female: Female
+male: Male
+other: Other
+monday: Monday
```

I guess it's better like this, cuz date functions return that string.

```suggestion
Monday: Monday
```